modules/ROOT/pages/_partials/common-releasenotes.adoc

```diff
@@ -132,13 +132,13 @@ The version of Inter-Sync{nbsp}Gateway useable depends on the Sync Gateway versi
 ====
 Inter-Sync Gateway Replication versions available for use on an active node::
 
-* Pre-Sync Gateway 2.8 -- only Inter-Sync Gateway Replication version 1 is available
-* For Pre-Sync Gateway 2.8+ -- you may run Inter-Sync Gateway Replication version 2 and Inter-Sync Gateway Replication version 1 (deprecated)
+* Pre-Sync Gateway 2.8 -- only SG{nbsp}Replicate is available
+* For Pre-Sync Gateway 2.8+ -- you may run Inter-Sync Gateway Replication and SG{nbsp}Replicate (deprecated)
```

Comment:
The second line is confusing. "Pre-Sync Gateway 2.8+" ?? Did you mean "Sync Gateway 2.8" ?

Reply:
indeed I did